internal/nginx/nginx.go

```diff
@@ -202,6 +204,7 @@ func NewNginxController(nginxConfPath string, local bool) *Controller {
 		nginxConfdPath:        path.Join(nginxConfPath, "conf.d"),
 		nginxSecretsPath:      path.Join(nginxConfPath, "secrets"),
 		local:                 local,
+		nginxBinaryPath:       path.Join("/usr/sbin/", nginxBinary),
```

Comment:
why not to set the absolute path in `main.go`? similar to `nginxConfPath` parameter.